aws_ansible_groups_and_ec2s/meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
+  author: your name
```

Comment:
How is this main.yml used? Should we fill in the relevant info on name, description, etc?

Reply:
Thanks for bringing that up. When ansible-galaxy init's a new role, it creates default folders for you to fill in. I forgot to look through and edit the meta/main.yml file. I'll probably just put in the description of the role and the tags I have in there.